rust-src/concordium_base/src/curve_arithmetic/field_adapters.rs

```diff
@@ -0,0 +1,57 @@
+//! Wrapper types and blanket implementations serving as adapters from
+//! the `ff` crate `Field`.
+use ff;
```

Comment:
Could you also explain why we need this?

Is it because curve25519 and ristretto use `ff`?

Does ark not have ristretto?

Reply:
The Ristretto representation from the `curve25519-dalek` implements the `ff` traits. The crate is relatively widely used, so it could be useful for integrating with other elliptic curve implementations.

I'll add this to the documentation.

Reply:
> Does ark not have ristretto?

They have curve25519 implementation, but not specifically the Ristretto representation, as far as I remember. That implementation is less used than the dalek one. Maybe this is something to look at in the future.